website/docs/components/accordion/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Accordion
+description: An accordion allow users to expand or collapse multiple sections of content in a stack.
```

Comment:
[nit] I've been trying to avoid use of language like "allow" or "enable", this is probably something we should brainstorm on as a team and whether there are alternatives.

For context, I've been trying to stay away from this type of language because it leans on giving the user permission to do something which has a different tone than empowering them with the tools to do something. I'm not sure if I've come across a suitable substitute quite yet, but something for us to consider more broadly.

Reply:
I added this content in my branch too.

This is what I currently have:
```
title: Accordion
description: An accordion is a vertically stacked list of container-like toggles that reveal or hide associated sections of content.
caption: A list of toggles that reveal or hide associated sections of content.
```

Reply:
Thanks for the detailed context. I've updated the content to avoid using that language.